sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationManager.kt

```diff
@@ -144,4 +144,19 @@ interface ViewAnnotationManager {
    * Remove an instance of [OnViewAnnotationUpdatedListener].
    */
   fun removeOnViewAnnotationUpdatedListener(listener: OnViewAnnotationUpdatedListener)
+
+  /**
+   * Set view annotation update mode defined in [ViewAnnotationUpdateMode].
+   */
+  fun setViewAnnotationUpdateMode(mode: ViewAnnotationUpdateMode)
+
+  /**
+   * Static methods and variables.
+   */
+  companion object {
```

Comment:
`internal` ?

Reply:
it's useful to expose our default value to the users so better leave public